docs/rapidstart.md

```diff
@@ -0,0 +1,717 @@
+# Rapidstart \- Hello UC\!
+
+This document provides an end-to-end installation and basic testing of all current components of Unity Catalog. It allows developers to try out the major aspects of Unity Catalog on their local machine. This rapid start will also be the basis for the full Unity Catalog Docker.
+
+# Quick Start
+
+To start working with Unity Catalog in your development environment, start with the [Quickstart \- Hello UC\!](https://github.com/unitycatalog/unitycatalog?tab=readme-ov-file#quickstart---hello-uc)  These steps allow you to:
+
+* [Run the UC Server](https://github.com/unitycatalog/unitycatalog?tab=readme-ov-file#run-the-uc-server)  
+* [Operate on Delta tables with the CLI](https://github.com/unitycatalog/unitycatalog?tab=readme-ov-file#operate-on-delta-tables-with-the-cli)  
+* [Operate on Delta tables with DuckDB](https://github.com/unitycatalog/unitycatalog?tab=readme-ov-file#operate-on-delta-tables-with-duckdb)  
+* [Interact with the Unity Catalog tutorial](https://github.com/unitycatalog/unitycatalog?tab=readme-ov-file#interact-with-the-unity-catalog-ui)
+
+# Using Unity Catalog with Apache Spark™ and Delta Lake
+
+For Apache Spark™ and Delta Lake to work together with Unity Catalog, you will need Apache Spark 3.5.3 and Delta Lake 3.2.1.
+
+## Download and Build Unity Catalog Spark Connector
+
+The following steps are required to download and build the Unity Catalog Spark Connector.
+
+### Download Apache Spark™ 3.5.3
+
+Download Apache Spark 3.5.3 at [https://spark.apache.org/downloads.html](https://spark.apache.org/downloads.html) or using the following command.
+
+```bash
+curl -O https://archive.apache.org/dist/spark/spark-3.5.3/spark-3.5.3-bin-hadoop3.tgz
+tar xzf spark-3.5.3-bin-hadoop3.tgz
+```
+
+### \[Optional\] Configure server properties for cloud storage
+
+To have Unity Catalog work with cloud object storage as the storage location for tables, configure the `etc/conf/server.properties` to add configuration:  
+
+```bash
+## S3 Storage Config (Multiple configs can be added by incrementing the index)
+s3.bucketPath.0=<S3_BUCKET>
+s3.region.0=<S3_REGION>
+s3.awsRoleArn.0=<S3_ROLE>
+# Optional (If blank, it will use DefaultCredentialsProviderChain)
+s3.accessKey.0=<SECRET>
+s3.secretKey.0=<SECRET>
+
+## ADLS Storage Config (Multiple configs can be added by incrementing the index)
+adls.storageAccountName.0=<ADLS_STORAGEACCOUNTNAME>
+adls.tenantId.0=<ADLS_TENANTID>
+adls.clientId.0=<ADLS_CLIENTID>
+adls.clientSecret.0=<SECRET>
+
+## GCS Storage Config (Multiple configs can be added by incrementing the index)
+gcs.bucketPath.0=<GCS_BUCKET>
+# Optional (If blank, it will use Default Application chain to find credentials)
+gcs.jsonKeyFilePath.0=/path/to/<SECRET>/gcp-key-uc-testing.json
+```
+
+### \[Optional\] Restart the UC Server
```

Comment:
Not sure these backslashes are necessary, but shouldn't matter as long as the page renders properly